src/libraries/Hooks.sol

```diff
@@ -87,15 +84,13 @@ library Hooks {
         // If a hook contract is set, it must have at least 1 flag set, or have a dynamic fee
         return address(hook) == address(0)
             ? !fee.isDynamicFee()
-            : (uint160(address(hook)) >= ACCESS_LOCK_FLAG || fee.isDynamicFee());
+            : (uint160(address(hook)) >= NO_OP_FLAG || fee.isDynamicFee());
     }
 
     /// @notice performs a hook call using the given calldata on the given hook
     /// @return expectedSelector The selector that the hook is expected to return
     /// @return selector The selector that the hook actually returned
     function _callHook(IHooks self, bytes memory data) private returns (bytes4 expectedSelector, bytes4 selector) {
```

Comment:
think we can even do a nicer refactor here now where we just call the hook in each callsite function and change this shared function to `validateHookResponse()` 
i can do in a followup tho